templates/settings.html

```diff
@@ -79,6 +79,11 @@
 				<input type="hidden" value="off" name="hide_hls_notification">
 				<input type="checkbox" name="hide_hls_notification" {% if prefs.hide_hls_notification == "on" %}checked{% endif %}>
 			</div>
+			<div id="visit_reddit_confirmation">
```

Comment:
```suggestion
			<div id="disable_visit_reddit_confirmation">
```

(will require concomitant change in `static/style.css`)